lib/hashdiff/diff.rb

```diff
@@ -90,6 +93,8 @@ def self.diff(obj1, obj2, options = {}, &block)
 
     opts[:prefix] = [] if opts[:array_path] && opts[:prefix] == ''
 
+    opts[:ignore_keys] = [*opts[:ignore_keys]]
```

Comment:
It's not clear what this line does.

Reply:
As a single value can be passed (sym or string) the default [] will be overridden by this and thus it is necessary to convert single args to an array with the splat.

Reply:
Thanks for explanation, a comment would be very helpful.

Reply:
Or, we can tighten the rules by requiring that `ignore_keys` must be an array, unless it's an established convention in Ruby libs.

Reply:
Will add a comment, as I think it is nice to have the flexibility of passing sym or string and it's a common enough idiom to splat a param that can be single object or an array.